### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by pgScorpio** [jamulussoftware/jamulus#1078] [UI Related]
Who is responsible for the translations for the multiple languages ? And how can I suggest improvements ?
I find missing and inconsistent translations in Dutch, I would like to offer my help, but don't know where to start.

**Comment by ann0see** [comment]
I'm not sure if this was already answered, I'm tagging @jerogee here.

**Reply by jerogee** [reply]
@ann0see , thanks. @pgScorpio, great! I already included your corrections and suggestions from #1069 as part of a PR with translation updates (#1110).  If you have further improvements, you can describe them in a ticket, tag me, and I will add them to that PR so it gets in the upcoming release.

If you want to contribute directly, also welcome. For that, update the Dutch translation file below and create a PR:
```src/res/translation/translation_nl_NL.ts```
